logging/kit/client_interceptors.go

```diff
@@ -0,0 +1,54 @@
+package kit
+
+import (
+	"path"
+	"time"
+
+	"context"
+
+	"github.com/go-kit/kit/log"
+	"google.golang.org/grpc"
+)
+
+// UnaryClientInterceptor returns a new unary client interceptor that optionally logs the execution of external gRPC calls.
+func UnaryClientInterceptor(logger log.Logger, opts ...Option) grpc.UnaryClientInterceptor {
```

Comment:
I just tried to use it and I cannot see the option that would allow me to inject custom fields here like `request-id` to the logged message ): I think it would be nice to add! It is exactly the same for logrus and zap but would nice to add I guess (: at some point in another PR!

Reply:
As commented here: https://github.com/grpc-ecosystem/go-grpc-middleware/pull/223/files#r341806517